### ISSUE TITLE
📎 Change logo alt text in main README

### ISSUE THREAD

**Issue by smhmd** [romefrontend/rome#1002]
[![image](https://user-images.githubusercontent.com/46059092/89723464-b3310700-d9ee-11ea-9d72-660e5df0173d.png)](https://github.com/romefrontend/rome/blob/main/README.md#L2)

The alt text should be changed. I liked the old logo a lot, so I couldn't come up with a proper alt text. `alt="A boring yellow gate. This used to be this badass ancient Greek spartan helmet"`, see?

**Comment by cyborg-z** [comment]
@smhmd maybe something along the lines of "An ancient roman arch colored yellow and white"

**Comment by sebmck** [comment]
Fixed in #1014, thanks @smhmd!
